app/Console/Commands/CreateMasterVoucherLogReport.php

```diff
@@ -73,16 +75,19 @@ class CreateMasterVoucherLogReport extends Command
      */
     private $report = <<<EOD
 SELECT
+  vouchers.code AS 'Voucher Number',
+  # To be determined, we don't know this yet.
+  '' AS 'Distributed to',
```

Comment:
Neat. Does that work?

Reply:
yep, you can put constant strings in columns, so en empty string is treated the same.